spec/features/users/create_spec.rb

```diff
@@ -94,19 +94,19 @@
   end
 
   context 'with external authentication', js: true do
+    let(:new_user) { Pages::NewUser.new }
```

Comment:
I'd highly advise that you append `_page` here. Otherwise I have to believe that `current_user` and `new_user` refer to the same kind of thing.